### PR TITLE
Add salaries

### PR DESCRIPTION
**Pull request by koffeinfrei**
png image 1587 x 1262 pixels - scaled 62

### CODE REVIEW

File: widgets/graph/graph.scss [hunk 1]
```diff
@@ -41,6 +41,7 @@ $tick-color:        rgba(0, 0, 0, 0.4);
   }
 
   .x_tick {
+    display: none; // we don't need to who the x-axis numbers as they don't tell anything meaningful
```
Comment (exul) [inline widgets/graph/graph.scss]: :cop: who -> show?

File: jobs/controllr.rb [hunk 2]
```diff
@@ -18,6 +18,10 @@ def update
   # working hours
   hours_worked = controllr.hours_worked(Date.today.month)
   send_event('hours-worked', { current: hours_worked })
+
+  # salaries per month
+  points = YAML.load_file('config/salaries.yml')[2012].map { |key, value| { x: key, y: value } }
```
Comment (exul) [inline jobs/controllr.rb]: why 2012 and not the current ones?

Comment (koffeinfrei) [inline jobs/controllr.rb]: Fixed while you were reviewing :smile:

**Comment by exul** [comment]
:shipit:  but with a press release please

**Pull request merged**